packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -460,11 +462,13 @@ export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldSt
 
   private _renderInput(): React.ReactElement<React.HTMLAttributes<HTMLInputElement>> {
     const inputProps = getNativeProps<React.HTMLAttributes<HTMLInputElement>>(this.props, inputProperties, ['defaultValue']);
-
+    let labelForAria = this.props['aria-labelledby'] ? this.props['aria-labelledby'] : undefined;
```

Comment:
This way would be a bit easier to read (also, it's probably better to call the variable `ariaLabelledBy`):
```
const ariaLabelledBy = this.props['aria-labelledby'] || (this.props.label ? this._labelId : undefined);
```